concrete-core/src/backends/core/private/crypto/bootstrap/fourier/buffers.rs

```diff
@@ -19,20 +18,18 @@ pub struct FftBuffers {
 }
 
 #[derive(Debug, Clone)]
-pub struct FourierBskBuffers<Scalar> {
+pub struct FourierBuffers<Scalar> {
```

Comment:
Could we include some documentation about this structure? It is not clear to me what it is, and where it is used, and why we needed it. I feel like the naming is not very explicit.

Reply:
@agnesLeroy @aPere3 I suppose this calls for another issue?

Reply:
@orelcosseron @damienligier yes I'll open an issue for this, let's keep this PR just for the external product, it's already really big.

Reply:
I'll keep this conversation unresolved until the issue is opened.